src/main/java/ch/njol/skript/util/Date.java

```diff
@@ -67,18 +65,34 @@ public String toString() {
 	}
 	
 	/**
+	 * Get the timestamp of this date
+	 *
 	 * @return The timestamp in milliseconds
 	 */
 	public long getTimestamp() {
 		return timestamp;
 	}
 	
-	public void add(final Timespan span) {
+	/**
+	 * Add a {@link Timespan} to this date
+	 *
+	 * @param span Timespan to add
+	 * @return This date with the added timespan
+	 */
+	public Date add(final Timespan span) {
 		timestamp += span.getMilliSeconds();
+		return this;

```

Comment:
usually, `add`/`subtract` methods that return an instance don't modify the existing instance because you can end up with confusing code like this:
```
Date today = new Date();
Timespan oneDay = ...;
Date tomorrow = today.add(oneDay);
System.out.println(tomorrow == today); // confusingly, prints true
```

Reply:
Ah good call. What would suggest for this, maybe return a new date?

Reply:
usually would do a method called `plus()` that returns a new date of the new time. if you want to make it really pretty, you could have a `Date#now` which returns a date of the current time then it would look like
```
Timespan oneYear = ...;
Date.now().plus(oneYear);
```
which is beautiful and very understandable

Reply:
oooo yes. Ok I like this. Ill work on it and add it shortly. Thank you!